app/templates/gulpfile.js

```diff
@@ -231,6 +231,91 @@ gulp.task('init-local', function() {
 });
 
 /**
+ * Creates a sample process.json
+ */
+gulp.task('init-process', function() {
+  //copy from slushfile - with modifications to use config instead
+  log('Creating process.json sample document with values drawn from gulp.config.js');
+  try {
+    var configJSON = pm2JSON();
+
+    var configString = JSON.stringify(configJSON, null, 2) + '\n';
+    fs.writeFileSync('process.json', configString, {
+      encoding: 'utf8'
+    });
+  } catch (e) {
+    console.log('failed to write process.json: ' + e.message);
+  }
+});
```

Comment:
We discussed adding a fixed process.json/boot.json to templates, so you don't really need this, particularly since we noticed only app name will be different for each app..